python/pipeline/stack.py

```diff
@@ -6,6 +6,7 @@
 import scanreader
 from scipy import signal
 from scipy import ndimage
+from scipy.optimize import curve_fit
```

Comment:
I'm gonna be annoying and ask you to import optimize rather than curve_fit directly so `from scipy import optimize` and then use `optimize.curve_fit`, the advantage is that tells me curve_fit is coming from optimize rather than being a function defined by someone (http://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
Not annoying at all. I changed it to fit that style.